src/core/mount-setup.c

```diff
@@ -406,6 +407,13 @@ int mount_setup(bool loaded_policy) {
         mkdir_label("/run/systemd", 0755);
         mkdir_label("/run/systemd/system", 0755);
         mkdir_label("/run/systemd/inaccessible", 0000);
+        /* Set up inaccessible items */
+        mknod("/run/systemd/inaccessible/reg", S_IFREG | 0000, 0);
+        mkdir_label("/run/systemd/inaccessible/dir", 0000);
+        mknod("/run/systemd/inaccessible/chr", S_IFCHR | 0000, makedev(0, 0));
+        mknod("/run/systemd/inaccessible/blk", S_IFBLK | 0000, makedev(0, 0));
+        mkfifo("/run/systemd/inaccessible/fifo", 0000);
+        mknod("/run/systemd/inaccessible/sock", S_IFSOCK | 0000, 0);
```

Comment:
nitpick: given that we knowingly ignore the return value of these calls, they should probably be prefixed with `(void)` to explicitly cast the return value away. This doesn't matter much, but is recommended in the CODING_STYLE docs. And yeah, the existing code around this doesn't do this either, but it really should too...